engines/router/missionctl/handlers/http_handler.go

```diff
@@ -187,7 +191,6 @@ func (h *httpHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	for key := range resp.Header() {
 		rw.Header().Set(key, resp.Header().Get(key))
 	}
-	rw.Header().Set(turingReqIDHeaderKey, turingReqID)
```

Comment:
We should still set the `turingReqIDHeaderKey` in the response header as if not, it will break the current behavior.

Reply:
Oops good catch; I've added it back again